app/models/address.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class Address < ApplicationRecord
+  has_one :organization, dependent: :restrict_with_exception
```

Comment:
Wouldn't it more be like a nullify?

Reply:
i used what creditor detail used

Reply:
but i can change for sure

Reply:
well I had to change creditor_detail in my admin pr

Reply:
so should i do :nullify?

Reply:
problem there is org currently has not null on address column

Reply:
so that would cause problems i'd say

Reply:
ah ok